### PR TITLE
Have a command that toggles throw ball with mouse on

### PR DESCRIPTION
**Pull request by Harry-Hopkinson**
"Throw ball with mouse" command that toggles the throw ball with mouse option on or off. For #375

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -84,7 +84,7 @@
                 "icon": "media/dog.svg"
             },
             {
-                "command": "vscode-pets.throw-with-mouse",
+                "command": "vscode-pets.throwBallWithMouse",
```
Comment (tonybaloney) [inline package.json:87]: Why did you change the command ID to camelCase?

Comment (Harry-Hopkinson) [reply package.json:87]: Sorry it had the same name as a variable in extension.ts so I presumed that it had to be named the same.

**Comment by tonybaloney** [comment]
I have had to force a bunch of changes to this repository to remove the cat assets (and previous copies). GitHub automatically closed the PR. You may need to create a new fork, before submitting future changes.
